### PR TITLE
Set docker summaries for all extensions.

### PR DESCRIPTION
**Pull request by sebastienros**
## Description

Saw that it was missing on SqlServer during Damian's demo.

## Checklist

- Is this feature complete?
  - [x] Yes. Ready to ship.
  - [ ] No. Follow-up changes expected.
- Are you including unit tests for the changes and scenario tests if relevant?
  - [ ] Yes
  - [x] No
- Did you add public API?
  - [ ] Yes
    - If yes, did you have an API Review for it?
      - [ ] Yes
      - [ ] No
    - Did you add `<remarks />` and `<code />` elements on your triple slash comments?
      - [ ] Yes
      - [ ] No
  - [x] No
- Does the change make any security assumptions or guarantees?
  - [ ] Yes
    - If yes, have you done a threat model and had a security review?
      - [ ] Yes
      - [ ] No
  - [x] No
- Does the change require an update in our Aspire docs?
  - [ ] Yes
    - Link to aspire-docs issue: 
  - [x] No

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/aspire/pull/6323)

### CODE REVIEW

File: src/Aspire.Hosting.Azure.CosmosDB/AzureCosmosDBExtensions.cs [hunk 1]
```diff
@@ -98,7 +98,8 @@ public static IResourceBuilder<AzureCosmosDBResource> AddAzureCosmosDB(this IDis
 
     /// <summary>
     /// Configures an Azure Cosmos DB resource to be emulated using the Azure Cosmos DB emulator with the NoSQL API. This resource requires an <see cref="AzureCosmosDBResource"/> to be added to the application model.
-    /// For more information on the Azure Cosmos DB emulator, see <a href="https://learn.microsoft.com/azure/cosmos-db/emulator#authentication"></a>
+    /// For more information on the Azure Cosmos DB emulator, see <a href="https://learn.microsoft.com/azure/cosmos-db/emulator#authentication"></a>.
+    /// This version of the package defaults to the <inheritdoc cref="CosmosDBEmulatorContainerImageTags.Tag"/> tag of the <inheritdoc cref="CosmosDBEmulatorContainerImageTags.Registry"/>/<inheritdoc cref="CosmosDBEmulatorContainerImageTags.Image"/> container image.
```
Comment (eerhardt) [inline src/Aspire.Hosting.Azure.CosmosDB/AzureCosmosDBExtensions.cs:102]: This seems more like `<remarks>` information, and not necessarily `<summary>` information.

Thoughts @IEvangelist?

File: src/Aspire.Hosting.Redis/RedisContainerImageTags.cs [hunk 2]
```diff
@@ -21,7 +21,7 @@ internal static class RedisContainerImageTags
     public const string RedisCommanderImage = "rediscommander/redis-commander";
 
     /// <summary>latest</summary>
-    public const string RedisCommanderTag = "latest";
+    public const string RedisCommanderTag = "latest"; // There aren't better tag than 'latest' which is 3 years old.
```
Comment (eerhardt) [inline src/Aspire.Hosting.Redis/RedisContainerImageTags.cs:24]:
```suggestion
    public const string RedisCommanderTag = "latest"; // There isn't a better tag than 'latest' which is 3 years old.
```

File: src/Aspire.Hosting.Azure.CosmosDB/AzureCosmosDBExtensions.cs (hunk 1 above)
Comment (sebastienros) [reply src/Aspire.Hosting.Azure.CosmosDB/AzureCosmosDBExtensions.cs:102]: I tried to keep the existing pattern, but I would agree with that change.

Comment (eerhardt) [reply src/Aspire.Hosting.Azure.CosmosDB/AzureCosmosDBExtensions.cs:102]: Yeah, it looks a little mixed. Some places put it in the remarks today.

File: src/Aspire.Hosting.MySql/MySqlBuilderExtensions.cs [hunk 3]
```diff
@@ -79,9 +82,12 @@ public static IResourceBuilder<MySqlDatabaseResource> AddDatabase(this IResource
...
 
     /// <summary>
-    /// Adds a phpMyAdmin administration and development platform for MySql to the application model. This version of the package defaults to the <inheritdoc cref="MySqlContainerImageTags.PhpMyAdminTag"/> tag of the <inheritdoc cref="MySqlContainerImageTags.PhpMyAdminImage"/> container image.
+    /// Adds a phpMyAdmin administration and development platform for MySql to the application model.
     /// </summary>
-    /// <param name="builder">The MySql server resource builder.</param>
+    /// <remarks>
+    /// This version of the package defaults to the <inheritdoc cref="MySqlContainerImageTags.PhpMyAdminTag"/> tag of the <inheritdoc cref="MySqlContainerImageTags.PhpMyAdminImage"/> container image.
+    /// </remarks>
+    /// /// <param name="builder">The MySql server resource builder.</param>
```
Comment (eerhardt) [inline src/Aspire.Hosting.MySql/MySqlBuilderExtensions.cs:90]:
```suggestion
    /// <param name="builder">The MySql server resource builder.</param>
```

**Review by eerhardt — APPROVED** [review]
LGTM

**Comment by sebastienros** [comment]
@eerhardt As I was writing what validation I had just done to ensure it's working, I realized that moving it to `<remarks>` is actually breaking it. Because it will now "inherit" the remarks tags. So the `/// <summary>latest</summary>` on the tags now need to be in `/// <remarks>` too.

**Comment by eerhardt** [comment]
>  So the `/// <summary>latest</summary>` on the tags now need to be in `/// <remarks>` too.

You mean "instead of" right?

**Comment by sebastienros** [comment]
"too" because I changed summary to remarks, so the Tags also need the change. But yes. 
I applied the change on the backport PR, assuming we can then merge release/9.0 back to main. Not sure we need to update it in main in that case.

**Comment by davidfowl** [comment]
backport?

**Comment by eerhardt** [comment]
> backport?

https://github.com/dotnet/aspire/pull/6433

**Pull request merged**